src/main/java/edu/harvard/iq/dataverse/search/savedsearch/SavedSearchServiceBean.java

```diff
@@ -240,6 +253,37 @@ public JsonObjectBuilder makeLinksForSingleSavedSearch(DataverseRequest dvReq, S
         return response;
     }
 
+    public void unLinksForSingleSavedSearch(DataverseRequest dvReq, SavedSearch savedSearch) throws SearchException, CommandException {
```

Comment:
I don't love this method name. `unLinkForSingleSavedSearch` might be better. Or `removeLinkForSingleSavedSearch`?

I'm not even sure we need "single" in there. I probably added it back in the day for a reason but now it seems verbose. 🤷